pymc3/distributions/continuous.py

```diff
@@ -3786,6 +3759,31 @@ def logp(value, mu, sigma):
         )
 
 
+class InterpolatedRV(RandomVariable):
+    name = "interpolated"
+    ndim_supp = 0
+    ndims_params = [0, 0]
```

Comment:
I think `ndims_params` cannot be scalar (i.e., `0`), right? In that case it should be `ndims_params = [1, 1]`